### PR TITLE
Add name to readme

### PR DESCRIPTION
**Pull request by surajverma2587**
### Description

Joined a new team today and my task is to add my name to the readme file.

- [x] Add info to ticket
- [x] contact UX

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # Some App
 
-## Team Members
\ No newline at end of file
+## Team Members
+
+- Suraj Verma
```
Comment (surajverma2587) [inline README.md]: Can you make them uppercase

**Review by kepop1 — DISMISSED** [review]
Definitely going to approve that, but you might need fix the old conflict you have there ... 😉

File: README.md (hunk 1 above)
Comment (kepop1) [reply README.md]: Why not Kebab-Case ?

**Review by kepop1 — APPROVED** [review]
I'll merge this for now, but we need to talk about casing standards in the README 🤔

**Pull request merged**